### DISCUSSION TITLE
Display buildings in React w/ Mapbox

### DISCUSSION THREAD

**Discussion by ventoline** [visgl/deck.gl#8174] [Q&A]
Despite the many examples and discussions using buildings, I cant find a way to display buildings in React using Mapbox (react-map-gl). 
How is it done:  is it in the mapbox style? a composite layer (then is it a mapboxlayer or deck.gl layer?)?, an example with the syntax would be most appreciated 🙏

     ```
 <DeckGL
    initialViewState={INITIAL_VIEW_STATE}
    controller={true}
    layers={layers}
  >
    <Map 
          mapboxAccessToken={MAPBOX_ACCESS_TOKEN} 
          mapStyle="mapbox://styles/mapbox/light-v9"   />
</DeckGL> 
```

**Comment by Pessimistress** [comment]
https://deck.gl/examples/geojson-layer-polygons
https://deck.gl/examples/trips-layer
https://deck.gl/examples/mapbox

**Reply by ventoline** [reply]
Thank you.
Yes these are the examples I referred to. It is unclear how to add the buildings. For now i have added them in the map style on mapbox studio
![image](https://github.com/visgl/deck.gl/assets/3739983/9c5aec12-32e1-493e-8c8b-79cb5e479265)

**Reply by Pessimistress** [reply]
![image](https://github.com/visgl/deck.gl/assets/2059298/96c03d54-c5cf-4bdb-8ea4-bcf9b12fc9a4)

**Comment by ventoline** [comment]
Thank you for clarifying, i guess you are referring to this example? 
https://github.com/visgl/deck.gl/blob/8.9-release/examples/website/safegraph/app.js
i could never find where the "building" source was
```

  map.on('load', () => {
    map.addLayer({
      id: '3d-buildings',
      source: 'composite',
      'source-layer': 'building',
      filter: ['==', 'extrude', 'true'],
      type: 'fill-extrusion',
      minzoom: 14,
      paint: {
        'fill-extrusion-color': '#ccc',
        'fill-extrusion-height': ['get', 'height']
      }
    });

```
